website/docs/guides/dbt-ecosystem/sl-partner-integration-guide.md

```diff
@@ -0,0 +1,671 @@
+---
+title: "dbt Semantic Layer integration" 
+id: "sl-partner-integration-guide"
+description: Learn about partner integration guidelines, roadmap, and connectivity. 
+---
+
+# dbt Semantic Layer partner integration
+
+:::info Contact us
+
+This guide is for dbt Semantic Layer integration partners and explains integration guidelines, product roadmap, and connectivity.<br />
+
+To become a formal partner, integrate with the API, or have questions/feedback &mdash; **[contact us](mailto:semantic-layer@dbtlabs.com)** for more info.
+
+:::
+
+
+## Overview
+
+The dbt Semantic Layer allows users to dynamically generate and query datasets in downstream tools based on their dbt governed assets, such as metrics, models, and entities. It helps organizations manage complexities such as data, tools, and teams to make more efficient and trustworthy decisions.
+
+The rapid growth of different tools in the modern data stack has helped data professionals. The various tools help solve the diverse needs of different teams. The downside of this growth is the fragmentation of business logic across teams, tools, and workloads. 
+
+To solve this, the dbt Semantic Layer provides a platform where users can confidently leverage their data from within their tools. dbt Cloud's change management capabilities ensure that any user modifications made to core business constructs, like metrics or entities, are distributed into all the tools connected to the data platform.
+
+The dbt Semantic Layer is unopinionated about specific tools, use cases, or applications of data. The following summarizes some of the common use cases:
```

Comment:
I understand why we often use the word "opinionated" when discussing our software, but "unopinionated" is rather esoteric. A brief follow-up statement on why being unopinionated is beneficial to our Cloud users might be helpful.

Reply:
hey @callum-mcdata or @cafzal  not sure if have any insight on why the SL is unopinionated and why it's a good thing for users/partners?

Reply:
Good catch; the current wording is awkward. @mirnawong1 How about replacing line 26 with "The dbt Semantic Layer can be used for a variety of tools and applications of data. Here are some common use cases:"